projects/kit/components/select/select.style.less

```diff
@@ -16,18 +16,21 @@
     /* stylelint-enable */
 }
 
-.hosted {
+.t-hosted {
     display: block;
     border-radius: inherit;
 }
 
-.textfield {
+.t-textfield {
     border-radius: inherit;
     text-align: inherit;
 }
 
-.value {
-    display: flex;
+.t-value {
+    display: block;
```

Comment:
Unnecessary rule for `div`, however I'm not sure removing `flex` works here. Also adding vertical padding would probably break templates for `tuiTextfieldLabelOutside="true"` case where it's supposed to take over whole space.

Reply:
`Unnecessary rule for div` yes, sorry, forgot to clean up when testing

`tuiTextfieldLabelOutside="true"` But it's working

![image](https://user-images.githubusercontent.com/12021443/147331951-a7747a1e-bb34-43ae-a0f9-7ea857920014.png)